lib/solidus_bactracs/api/shipment_serializer.rb

```diff
@@ -102,6 +103,11 @@ def safe_rma_type
         rma_type ||= @config.evaluate_rma_type.call(@shipment)
       end
 
+      def get_rp_location
+        rp_location = @shipment.get_rma_rp_location if @shipment.get_rma_rp_location
+        rp_location ||= @config.default_rp_location.call(@shipment)
```

Comment:
is there a reason to memoize?  I'm worried that it would retain this value as long as the API class is instantiated, and thus all shipments would use the same logic (and be wrong)

```suggestion
        @config.default_rp_location.call(@shipment)
```